internal/olm/operator/registry/index/registry_pod.go

```diff
@@ -87,6 +89,12 @@ type RegistryPod struct { //nolint:maligned
 	pod *corev1.Pod
 
 	cfg *operator.Configuration
+
+	// FBC data
+	FBCContent  string
+	FBCDir      string
+	FBCFile     string
+	HasFBCLabel bool
```

Comment:
We should describe each one of these with a comment similar to all the other fields.